controller/model/posture_check_type_handlers.go

```diff
@@ -0,0 +1,85 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package model
+
+import (
+	"github.com/openziti/foundation/storage/boltz"
+)
+
+func NewPostureCheckTypeHandler(env Env) *PostureCheckTypeHandler {
+	handler := &PostureCheckTypeHandler{
+		baseHandler: newBaseHandler(env, env.GetStores().PostureCheckType),
+	}
+	handler.impl = handler
+	return handler
+}
+
+type PostureCheckTypeHandler struct {
+	baseHandler
+}
+
+func (handler *PostureCheckTypeHandler) newModelEntity() boltEntitySink {
+	return &PostureCheckType{}
+}
+
+func (handler *PostureCheckTypeHandler) Create(PostureCheckTypeModel *PostureCheckType) (string, error) {
+	return handler.createEntity(PostureCheckTypeModel)
+}
+
+func (handler *PostureCheckTypeHandler) Read(id string) (*PostureCheckType, error) {
+	modelEntity := &PostureCheckType{}
+	if err := handler.readEntity(id, modelEntity); err != nil {
+		return nil, err
+	}
+	return modelEntity, nil
+}
+
+func (handler *PostureCheckTypeHandler) ReadByIdOrName(idOrName string) (*PostureCheckType, error) {
```

Comment:
i came to really dislike this pattern of "idOrName" at ${two.jobs.ago} but if this is a ship that sailed - it's probably too late. i'd rather not infer idOrName this way if we can avoid it though

Reply:
I used an old template to generate the code. We no longer support IdOrName in the API.